DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Centre/Reports/ReportsViewModel.cs

```diff
@@ -42,4 +45,14 @@ public ActivityDataRowModel(PeriodOfActivity periodOfActivity, string format)
         public int Evaluations { get; set; }
         public int Registrations { get; set; }
     }
+
+    public class ActivityFilterModel

```

Comment:
Wonder if it's worth separating this file into several classes?

Reply:
I think yes as it's become more baroque, but I'd like to deal with that refactor in the middleware/edit branch.

Reply:
Sounds good!